packages/client/src/components/form/FormFieldGenerator.tsx

```diff
@@ -869,11 +914,47 @@ class FormSectionComponent extends React.Component<Props> {
     })
   }
 
+  getOnClickHandle = (
+    fields: IFormField[],
+    field: IButtonFormField,
+    setFieldValue: (fieldName: string, value: any) => void,
+    setFieldError: (fieldName: string, message: string) => void
+  ) => {
+    const trigger = fields.find((f) => f.name === field.options.trigger)
+    if (trigger && isFieldHttp(trigger)) {
+      return () => {
+        const {
+          options: { url, ...requestOptions }
+        } = trigger
+        const fieldsDependentOnTrigger = getDependentFields(fields, trigger)
+        fetch(url, requestOptions)
+          .then((res) => res.text())
```

Comment:
We need to have JSON responses here as well, so objects stored in the Formik form state